src/ast/passes/codegen_llvm.cpp

```diff
@@ -12,6 +12,7 @@
 #include <llvm/IR/LLVMContext.h>
 #include <llvm/IR/LegacyPassManager.h>
 #include <llvm/IR/Metadata.h>
+#include <llvm/Passes/PassBuilder.h>
```

Comment:
The CI problem seems to stem from this include. Changing it to:
```suggestion
#if LLVM_VERSION_MAJOR >= 14
#include <llvm/Passes/PassBuilder.h>
#endif
```
makes the CI pass.

Reply:
It's surprising that `llvm/Passes/PassBuilder.h` may cause differences. I have checked llvm 9 and do not find anything weird. Anyway, Added #if.